Sources/SwiftParser/IncrementalParseTransition.swift

```diff
@@ -172,15 +172,15 @@ struct IncrementalParseLookup {
       return false
     }
 
-    let nodeAffectRange = ByteSourceRange(
-      offset: node.position.utf8Offset,
-      length: nodeAffectRangeLength
-    )
+    let nodeAffectRange =
+      AbsolutePosition(
+        utf8Offset: node.position.utf8Offset
+      )..<AbsolutePosition(utf8Offset: node.position.utf8Offset + nodeAffectRangeLength)
```

Comment:
I think we can use `Range<AbsolutePosition>(start: node.position.utf8Offset, length: nodeAffectRangeLength)` here (or it’s called something like that).